util.go

```diff
@@ -118,6 +118,7 @@ func getCGroupCPUCore() (float64, error) {
 		return 0, err
 	}
 
+	// No resource limit value is configured, the value is -1
```

Comment:
It's possible that `getCGroupCPUCore` returns core=-1 and err =nil?

Reply:
No, because the defined cpuQuota is of uint64 type, when the read file value is negative, `strconv.ParseUint` will report an error, and eventually `strconv.parseInt` will return '0, nil'.

Reply:
Thank for your reply. Could we delete this variable `cpuCore` printout? Bcz It must equal 0.
https://github.com/mosn/holmes/pull/137/files#:~:text=h.Warnf(%22%5BHolmes%5D%20get%20CGroup%20CPU%20core%20failed%2C%20will%20try%20to%20obtain%20cpu%20in%20other%20ways.%20CPU%20core%3A%20%25v%2C%20error%3A%20%25v%22%2C%20cpuCore%2C%20err)

Reply:
Ok, so does this warn log need to be deleted? It is better to suggest or remind.

Reply:
I suggest still printing warn log, just delete `cpuCore` in the log.

Reply:
done.

Reply:
so, this comment is outdated? since no `-1` return value now.
I think we can remove it simply, since this comment is good enough: https://github.com/mosn/holmes/pull/137/files#diff-21f064a19eaebe49f3b7904fe21e5ea1efd58fef8e08b0a64a4e3799a12aa6fcR708